php/php.editor/tools/ASTPHP5Parser.cup

```diff
@@ -2625,10 +2625,9 @@ fully_qualified_class_name:className
     RESULT = className;
 :}
 
-| T_ARRAY:array
+| T_ARRAY:name
 {:
-    Identifier classId = new Identifier(arrayleft, arrayright, "array");
-    RESULT = classId;
+    RESULT = NamespaceName.create(nameleft, nameright, name);
```

Comment:
@junichi11 Is this really correct? I am sorry, but I simply cannot recall it... (yes, it is a long time ago, already :)

Thanks.

Reply:
`string` is recognized as a namespace name with #4037
https://github.com/apache/netbeans/blob/1d85a3e854fafef563a3b0986b88f4e9613a1d85/php/php.editor/tools/ASTPHP5Parser.cup#L4323-L4348

So, this should be OK, I think.
If we come across a problem, let's fix it. Thanks!

Reply:
Thanks!